float64 acceleration      # Acceleration between 0.0 - 1.0 (low - high)
---